website/snippets/_enterprise-permissions-table.md

```diff
@@ -31,31 +31,31 @@ Account roles enable you to manage the dbt Cloud account and manage the account
 | Webhooks                |     W         |               |                 |                |        |
 
 #### Project permissions for account roles
-
+ 
 |Project-level permission | Account Admin | Billing admin | Project creator | Security admin | Viewer | 
 |:-------------------------|:-------------:|:-------------:|:---------------:|:--------------:|:------:| 
 | Connections             |       W       |               |       W         |                |   R    |
-| Credentials             |       W       |               |       W         |                |   R    |
+| Environment credentials             |       W       |               |       W         |                |   R    |
 | Custom env. variables   |       W       |               |       W         |                |   R    |
-| dbt adapters            |       W       |               |       W         |                |   R    |
+| Data platform config            |       W       |               |       W         |                |   R    |
 | Develop (IDE or dbt Cloud CLI)           |       W       |               |       W         |                |        |
 | Environments            |       W       |               |       W         |                |   R    |
 | Jobs                    |       W       |               |       W         |                |   R    |
-| Metadata                |       R       |               |       R         |                |   R    |
+| Metadata                 |       R       |               |       R         |                |   R    |
```

Comment:
what does metadata mean here?

Reply:
no worries! asked here https://dbt-labs.slack.com/archives/C02N953LRDF/p1720515202761889?thread_ts=1720176389.355509&cid=C02N953LRDF

Reply:
hey @gcs272 , Topher confirmed that essentially 'metadata' here means permissions query the metadata graphql api